plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/util/MinioClient.java

```diff
@@ -130,16 +134,23 @@ public void onFailure(Throwable t)
     public List<String> listObjects(String bucket, String path)
     {
         try {
-            return stream(client.listObjects(bucket, path)).map(
-                    result -> {
-                        try {
-                            return result.get().objectName();
-                        }
-                        catch (Exception e) {
-                            throw new RuntimeException(e);
-                        }
-                    }
-            ).collect(toImmutableList());
+            return stream(client.listObjects(
+                    ListObjectsArgs.builder()
+                            .bucket(bucket)
+                            .prefix(path)
+                            .recursive(true)
+                            .useUrlEncodingType(false)
+                            .build()))
+                    .map(
+                            result -> {
+                                try {
+                                    return result.get().objectName();
+                                }
+                                catch (Exception e) {
+                                    throw new RuntimeException(e);
+                                }
+                            }
+                    ).collect(toImmutableList());
```

Comment:
`)` should be on prev line

(why didn't checkstyle catch this?)

Reply:
> why didn't checkstyle catch this?
I don't know.